docs/build.md

```diff
@@ -29,33 +29,68 @@
 # Building Triton
 
 Triton is built using the [build.py](../build.py) script. The build.py
-script supports both a Docker build and a non-Docker build.
+script currently supports building for the following platforms. See
+[Building on Unsupported
+Platforms](#building-on-unsupported-platforms) if you are
+attempting to build Triton on a platform that is not listed here.
 
-* [Build using Docker](#building-triton-with-docker) and the
-  TensorFlow and PyTorch containers from [NVIDIA GPU Cloud
+* [Ubuntu 20.04, x86-64](#ubuntu)
+
+* [Jetpack 4.x, NVIDIA Jetson (Xavier, Nano, TX2)](#jetpack)
+
+* [Windows 10, x86-64](#windows)
+
+## <a name="ubuntu"></a>Building for Ubuntu 20.04
+
+For Ubuntu-20.04 build.py supports both a Docker build and a
+non-Docker build.
+
+* [Build using Docker](#ubuntu-docker) and the TensorFlow and PyTorch
+  Docker images from [NVIDIA GPU Cloud
   (NGC)](https://ngc.nvidia.com>).
 
-* [Build without Docker](#building-triton-without-docker).
+* [Build without Docker](#ubuntu-without-docker).
 
-## Building Triton with Docker
+### <a name="ubuntu-docker"></a>Building with Docker
 
 The easiest way to build Triton is to use Docker. The result of the
 build will be a Docker image called *tritonserver* that will contain
 the tritonserver executable in /opt/tritonserver/bin and the required
 shared libraries in /opt/tritonserver/lib. The backends built for
 Triton will be in /opt/tritonserver/backends (note that as of the
-20.11 release the TensorRT backend is still included in the core of
+21.02 release the TensorRT backend is still included in the core of
 Triton and so does not appear in /opt/tritonserver/backends).
 
-Building with Docker ensures that all the correct CUDA, cudnn,
-TensorRT and other dependencies are handled for you. A Docker build is
-the default when using build.py.
+The first step for any build is to checkout the
+[triton-inference-server/server](https://github.com/triton-inference-server/server)
+repo branch for the release you are interested in building (or the
+master branch to build from the development branch). Then run build.py
+as described below. The build.py script performs these steps when
+building with Docker.
+
+* Fetch the appropriate \<xx.yy\>-py3-min image from NGC. The *min*
+  image is a minimal/base image that contains the CUDA, cuDNN,
+  TensorRT and other dependencies that are required to build Triton.
+
+* Create a *tritonserver_buildbase* Docker image that adds additional
+  build dependencies to the *min* image.
+
+* Run build.py within the *tritonserver_buildbase* image to actually
+  build Triton. See [Build without Docker](#ubuntu-without-docker) for
```

Comment:
Should we clarify that only Triton server and the backends are built by build.py (and not client libraries/ test utilities)?

Reply:
I added a sentence at the beginning.